Square_Root_Decomposition/Square_Root_Decomposition.cpp

```diff
@@ -0,0 +1,62 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+void update(int index,int value,int *blocks,int a[],int n){
+	blocks[(int)(index/sqrt(n))] += value-a[index];
+	a[index]=value;
+}
+//assuming 0 based indexing
```

Comment:
Add a new line above every function.
In your case add it above the comment for function.